pkg/kp/kv.go

```diff
@@ -37,6 +38,18 @@ func NewStore(opts Options) *Store {
 	return &Store{client: client}
 }
 
+// KVError encapsulates an error in a Store operation. Errors returned from the
+// Consul API cannot be exposed because they may contain the URL of the request,
+// which includes an ACL token as a query parameter.
```

Comment:
I wish we could at least extract the HTTP status code, but I think I'm okay without it so long as we can still execute `consul monitor`. Eventually though this really needs to get addressed